### PR TITLE
Added support for handling exdate as arrays

### PR DESCRIPTION
**Pull request by stephenbinns**
EXDATE can support values as an array in the format:

```
EXDATE;TZID=Asia/Dubai:20171102T174000,20171026T174000,20171019T174000,20
 171109T174000
```
See https://www.kanzaki.com/docs/ical/exdate.html for reference

### CODE REVIEW

**Comment by stephenbinns** [comment]
Not sure what the cause of the spec fail is appears to be 1.9.3 only - is CI working?

File: lib/icalendar/recurrence/schedule.rb [hunk 1]
```diff
@@ -76,13 +76,18 @@ def ice_cube_schedule
         end
 
         event.exdate.each do |exception_date|
-          exception_date = Time.parse(exception_date) if exception_date.is_a?(String)
-          schedule.add_exception_time(TimeUtil.to_time(exception_date))
+          if exception_date.is_a?(Icalendar::Values::Array)
```
Comment (jnraine) [inline lib/icalendar/recurrence/schedule.rb]: What do you think about combining this into a single case?

```ruby
event.exdate.each do |exception_date_or_dates|
  Array(exception_date_or_dates).each do |exception_date|
    schedule.add_exception_time(TimeUtil.to_time(exception_date))
  end
end
```

We'll gain support for multiple exception dates while keeping the code as straightforward as it was before.

File: spec/lib/recurrence_spec.rb [hunk 2]
```diff
@@ -52,9 +52,19 @@
     end
   end
 
+  context "event repeating every thursday with multiple exceptions" do
+    let(:event) { example_event :multiple_exception } # has exclusions in array form
```
Comment (jnraine) [inline spec/lib/recurrence_spec.rb]: Would you be able to add a case for a singular exception date too? Looks like I missed it when writing the original implementation.

**Comment by stephenbinns** [comment]
Closed in favor of #21

**Pull request closed without merging**